.github/DEVELOPMENT.md

```diff
@@ -105,32 +105,5 @@ the code base like `ttl` are allowed and encouraged.
 
 ## Releasing
 
-- [Set up your development environment](#getting-started).
-- Check the local workspace is up to date and has no uncommitted changes
-  ```bash
-  git fetch -a && git status
-  ```
-- Change version in `trino/__init__.py` to a new version, e.g. `0.123.0`.
-- Commit
-  ```bash
-  git commit -a -m "Bump version to 0.123.0"
-  ```
-- Create an annotated tag
-  ```bash
-  git tag -m "" 0.123.0
-  ```
-- Create release package and upload it to PyPI
-  ```bash
-  . .venv/bin/activate && \
-  pip install twine wheel setuptools && \
-  rm -rf dist/ && \
-  ./setup.py sdist bdist_wheel && \
-  twine upload dist/* && \
-  open https://pypi.org/project/trino/ && \
-  echo "Released!"
-  ```
-- Push the branch and the tag
-  ```bash
-  git push upstream master 0.123.0
-  ```
+- Manually trigger the release workflow which will auto-increment the version number and conduct a release to PyPi.
```

Comment:
not yet - no actual release has been performed with it yet - the upcoming one will be and then we can update it.

Reply:
Done